engine/cache/manager.go

```
@@ -429,9 +452,7 @@ func (m *manager) Import(ctx context.Context) error {
 // Close will block until the final export has finished or ctx is canceled.
 func (m *manager) Close(ctx context.Context) (rerr error) {
 	close(m.startCloseCh)
-	if m.stopCacheMountSync != nil {
-		rerr = m.stopCacheMountSync(ctx)
-	}
+	m.UploadCacheMounts(ctx)
```

is ignoring the error here intentional? Seems worth logging at least